src/Services/Filters.js

```diff
@@ -53,6 +51,17 @@ const filtersInitialState = {
   clusterRulesState: CLUSTER_RULES_INITIAL_STATE,
 };
 
+export const resetFilters = (filters, initialState, updateFilters) => {
+  const { limit, offset, sortIndex, sortDirection } = filters;
+  updateFilters({
+    ...initialState,
+    ...(limit !== undefined && { limit }),
+    ...(limit !== undefined && { offset }),
```

Comment:
I think there is a typo?
```suggestion
    ...(offset !== undefined && { offset }),
```

Reply:
Nope, not a typo. The ClusterRules table, which doesn't have pagination, returns `0` for `offset` when it calls this `resetFilters` function. So, I check for `limit === undefined` because then I know I'm on the ClusterRules page.